src/mirage_kv.mli

```diff
@@ -134,6 +134,13 @@ module type RO = sig
       The result is [Error (`Value_expected k)] if [k] refers to a
       dictionary in [t]. *)
 
+  val get_partial: t -> key -> offset:int -> length:int -> (string, error) result Lwt.t
+  (** [get_partial t k ~offset ~length] is the [length] bytes wide value
+     bound at [offset] of [k] in [t].
+
+      The result is [Error (`Value_expected k)] if [k] refers to a
+     dictionary in [t]. *)
```

Comment:
what is the expected result if `foo` is bound to `ABCD` and `get_partial foo ~offset:1 ~length:4` is requested? an error or "BCD" (i.e. short reads?)

Reply:
You're totally right I expect the length to be what I can read at most, and it can be less than that, I'll explain that in the .mli.

Reply:
Oops, please ignore. Thanks for describing that scenario as well :)